server/protocols/jmap-integration-testing/cassandra-jmap-integration-testing/src/test/java/org/apache/james/jmap/cassandra/CassandraDeletedMessageVaultTest.java

```diff
@@ -34,6 +34,8 @@
 import org.apache.james.vault.MailRepositoryDeletedMessageVault;
 import org.apache.james.webadmin.WebAdminConfiguration;
 import org.junit.Rule;
+import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
```

Comment:
Euh no that's not how we should do it.

Let's do the swap later on, on a product per product basis.

Please create a new test class using the override of the BlobStoreDeletedMessageVaultModule - in that class, you can break whatever you want.

But do not cut features by switching to a partial implementation, backed only by memory!